items/SILEX.json

```diff
@@ -1,10 +1,9 @@
 {
   "itemid": "minecraft:quartz",
-  "displayname": "§f§f§fSharp Rock",
```

Comment:
You still dont need to remove these again and again. They will be on every new item bcs its from hypixel.

Reply:
"&f&f" is only shown on the auction house for some reason. That is the only reason why I remove them. I don't remove things like "&7&7" etc. in actual lores, as that is a part of the item. I know it makes no difference visibly, but technically the default doesn't have that. I certainly won't fix them otherwise, but I might as well if I am already editing the file.